v2/AdminCore.Constants/EmployeeLocationConstants.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace AdminCore.Constants
+{
+  public class EmployeeLocationConstants
+  {
+    public static string OnHoliday = "On Holiday";
```

Comment:
Small thing, change this to AnnualLeave for consistency with the rest of the project